.ci/docker/common/install_cudnn.sh

```diff
@@ -5,9 +5,9 @@ if [[ -n "${CUDNN_VERSION}" ]]; then
     mkdir tmp_cudnn
     pushd tmp_cudnn
     if [[ ${CUDA_VERSION:0:2} == "12" ]]; then
-        CUDNN_NAME="cudnn-linux-x86_64-9.1.0.70_cuda12-archive"
+        CUDNN_NAME="cudnn-linux-x86_64-9.5.0.50_cuda12-archive"
     elif [[ ${CUDA_VERSION:0:2} == "11" ]]; then
-        CUDNN_NAME="cudnn-linux-x86_64-9.1.0.70_cuda11-archive"
```

Comment:
Should we switch to the earlier pattern that different CUDA using different cudnn version? i.e. just updating the cudnn version for cuda 12.4.1?